tests/nwchem_tests/test_dft_xc.py

```diff
@@ -0,0 +1,555 @@
+
+@using_nwchem
+def test_33_cft97():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'cft97'
+    })
+    val = qcdb.energy('nwc-cft97')
+    check_cft97(val)
+
+def check_acm(return_value):
+    acm = -76.397101204907
+    assert compare_values(acm, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft acm')
+
+@using_nwchem
+def test_34_acm():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'acm'
+    })
+    val = qcdb.energy('nwc-acm')
+    check_acm(val)
+
+def check_optx(return_value):
+    optx = -76.055691019559
+    assert compare_values(optx, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft optx')
+
+@using_nwchem
+def test_35_optx():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'optx'
+    })
+    val = qcdb.energy('nwc-optx')
+    check_optx(val)
+
+def check_b98(return_value):
+    b98 = -76.394526199939
+    assert compare_values(b98, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft b98')
+
+@using_nwchem
+def test_36_b98():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'becke98'
+    })
+    val = qcdb.energy('nwc-b98')
+    check_b98(val)
+
+def check_xtpss03(return_value):
+    xtpss03 = -76.090088650605
+    assert compare_values(xtpss03, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft xtpss03')
+
+@using_nwchem
+def test_37_xtpss03():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'xtpss03'
+    })
+    val = qcdb.energy('nwc-xtpss03')
+    check_xtpss03(val)
+
+def check_bb1k(return_value):
+    bb1k = -76.387274033902
+    assert compare_values(bb1k, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft bb1k')
+
+@using_nwchem
+def test_38_bb1k():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'bb1k'
+    })
+    val = qcdb.energy('nwc-bb1k')
+    check_bb1k(val)
+
+def check_vs98(return_value):
+    vs98 = -76.441813521942
+    assert compare_values(vs98, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft vs98')
+
+@using_nwchem
+def test_39_vs98():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'vs98'
+    })
+    val = qcdb.energy('nwc-vs98')
+    check_vs98(val)
+
+def check_m06_l(return_value):
+    m06_l = -76.413915672071
+    assert compare_values(m06_l, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft m06-l')
+
+@using_nwchem
+def test_40_m06_l():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'm06-l'
+    })
+    val = qcdb.energy('nwc-m06-l')
+    check_m06_l(val)
+
+def check_hcth147(return_value):
+    hcth147 = -76.406989374352
+    assert compare_values(hcth147, qcdb.get_variable('DFT TOTAL ENERGY'), 5, 'dft hcth147')
+
+@using_nwchem
+def test_41_hcth147():
+    qcdb.set_options({
+        'basis' : 'cc-pvdz',
+        'nwchem_dft__xc': 'hcth147'
+    })
+    val = qcdb.energy('nwc-hcth147')
+    check_hcth147(val)
```

Comment:
we'll probably parametrize this later. but great to have all these values checked.